integration/kubernetes/k8s-sandbox-bindmount.bats

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2022 Apple Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+SYSCONFIG_FILE="/etc/kata-containers/configuration.toml"
+QEMU_CFG="/usr/share/defaults/kata-containers/configuration-qemu.toml"
+CLH_CFG="/usr/share/defaults/kata-containers/configuration-clh.toml"
+DEFAULT_CFG="/usr/share/defaults/kata-containers/configuration.toml"
+
+TEST_FILE="test-file"
+TEST_FILE_HOST_PATH="/usr/share/kata-containers/$TEST_FILE"
+
+setup() {
+	pod_name="qos-test"
+	sudo mkdir -p $(dirname "${SYSCONFIG_FILE}")
+
+	if [ "$KATA_HYPERVISOR} = "qemu" ];  then
+		cfg="${QEMU_CFIG}"
+	elsif [ "${HYPERVISOR}" = "cloud-hypervisor"]; then
+		cfg="${CLH_CFG}"
+	else
+		cfg="${DEFAULT_CFG}"
+	fi
+
+	[ -z "${cfg}" ] && echo "Configuration file not found" && false
+
+	cp -a "${cfg}" "${SYSCONFIG_FILE}"
+
+        sudo sed -i 's|sandbox_bind_mounts.*|sandbox_bind_mounts=["$TEST_FILE_HOST_PATH"]|g' "${SYSCONFIG_FILE}"
+	echo "hello" > "${TEST_FILE_HOST_PATH}"
+}
+
+@test "Check capabilities of pod" {
+	[ "${KATA_HYPERVISOR}" == "firecracker" ] && skip "test not working see: ${fc_limitations}"
```

Comment:
I can't see where `fc_limitations` is set, but `grep` shows:

```bash
$ grep -r fc_limitations=
$ integration/kubernetes/k8s-ro-volume.bats:fc_limitations="https://github.com/kata-containers/documentation/issues/351"
integration/kubernetes/k8s-volume.bats:fc_limitations="https://github.com/kata-containers/documentation/issues/351"
integration/kubernetes/k8s-cpu-ns.bats:fc_limitations="https://github.com/kata-containers/documentation/issues/351"
integration/kubernetes/k8s-projected-volume.bats:fc_limitations="https://github.com/kata-containers/documentation/issues/351"
integration/kubernetes/k8s-credentials-secrets.bats:fc_limitations="https://github.com/kata-containers/documentation/issues/351"
```

We've got some duplication there but also:

- That issue is a Kata 1.x one.
- That issue is closed (admittedly by me but probably part of the cleanup work for Kata 1.x when we archived the repo).

Are the points on that old 1.x issue still valid I wonder? If so, I'd be tempted to raise a new issue in this repo or the main one with a more current status update for FC.